Mathlib/SetTheory/Cardinal/Finite.lean

```diff
@@ -224,13 +224,20 @@ theorem card_zmod (n : ℕ) : Nat.card (ZMod n) = n := by
 end Nat
 
 namespace Set
+variable {s : Set α}
 
 lemma card_singleton_prod (a : α) (t : Set β) : Nat.card ({a} ×ˢ t) = Nat.card t := by
   rw [singleton_prod, Nat.card_image_of_injective (Prod.mk.inj_left a)]
 
 lemma card_prod_singleton (s : Set α) (b : β) : Nat.card (s ×ˢ {b}) = Nat.card s := by
   rw [prod_singleton, Nat.card_image_of_injective (Prod.mk.inj_right b)]
 
+lemma Nonempty.natCard_pos (hs : s.Nonempty) (hs' : s.Finite) : 0 < Nat.card s := by
```

Comment:
Please also add the iff version, as in [Set.ncard_pos](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Card.html#Set.ncard_pos). It also looks like you can golf this proof using that one.